target/linux/ipq40xx/image/mikrotik.mk

```diff
@@ -19,6 +19,14 @@ define Device/mikrotik_hap-ac2
 endef
 TARGET_DEVICES += mikrotik_hap-ac2
 
+define Device/mikrotik_lhg-5-ac
+	$(call Device/mikrotik_nor)
+	DEVICE_MODEL := LHG 5 ac (RBLHGG-5acD)
```

Comment:
Please remove the brackets and use DEVICE_ALT0_VENDOR and *_MODEL instead.

Reply:
This is just the product code for the device and not an alternative model? I can also just remove it completely?

Reply:
This here is exactly what the DEVICE_ALT* syntax was developed for. It's meant for cases where a single device has two different names, i.e. to get rid of the brackets in DEVICE_MODEL.

Everything else like using the same recipe for different devices was actually an abuse of this initial concept.

Reply:
Hi,

As I understood it (please correct me if I'm wrong), the ALT* syntax was used for identical devices with different manufacturer and/or name, like:
```
  DEVICE_VENDOR := AirTight Networks
  DEVICE_MODEL := C-75
  DEVICE_ALT0_VENDOR := Mojo Networks
  DEVICE_ALT0_MODEL := C-75
  DEVICE_ALT1_VENDOR := WatchGuard
  DEVICE_ALT1_MODEL := AP320
```
or different versions with identical hardware:
```
  DEVICE_MODEL := TL-WR941ND
  DEVICE_VARIANT := v2/v3
  DEVICE_ALT0_VENDOR := TP-Link
  DEVICE_ALT0_MODEL := TL-WR941N
  DEVICE_ALT0_VARIANT := v2/v3
```

I am not sure if this is the case here, with MikroTik devices. This brand [most often] uses a short "commercial" name for their devices and also assigns them a "product code". For instance, "SXTsq 5 ac" (commercial) and "RBSXTsqG-5acD" (product code) (see https://mikrotik.com/product/sxtsq_5_ac). Sometimes, however, the same commercial name is used for devices with equivalent features but totally different hardware (e.g., ath79 "wAP AC (RBwAPG-5Hac**T**2HnD)" and ipq40xx "wAP AC RBwAPG-5Hac**D**2HnD"

In my opinion, it is good to use both commercial and product code here, and also in the DTS file, so that e.g., /proc/cpuinfo displays it.

Just my 2 cents.